### DISCUSSION TITLE
Is Zooi ok?

### DISCUSSION THREAD

**Discussion by Drogo-man** [studio-minus/ppg-bugs-and-requests#2577] [General]
He hasn't really been active, nor been updating the game.
Is everything alright or what is happening?

**Comment by ATPGaming** [comment]
He might be busy with life, or he just forgot about us :(

**Comment by ghost** [comment]
On his profile you can see that he is contributing to private repos (which i assume is the madness combat game source code) and on the Studio Minus website there is a [blog](https://studiominus.nl/blog/index.html) where you can see progress on development

**Reply by Drogo-man** [reply]
> On his profile you can see that he is contributing to private repos (which i assume is the madness combat game source code) and on the Studio Minus website there is a [blog](https://studiominus.nl/blog/index.html) where you can see progress on development

Shit yeah, forgot about that. As you can see with my PFP, I'm very excited for it's release.
